jujugui/static/gui/src/app/components/error-boundary/error-boundary.js

```diff
@@ -0,0 +1,54 @@
+/* Copyright (C) 2018 Canonical Ltd. */
+'use strict';
+
+const React = require('react');
+const PropTypes = require('prop-types');
+
+require('./_error-boundary.scss');
+
+class errorBoundary extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {error: null};
+  }
+
+  componentDidCatch(error) {
+    this.setState({error});
+  }
+
+  render() {
+    const error = this.state.error;
+    if (error) {
+      return (
+        <div className="v1">
+          <div className="row">
+            <div className="col-12">
+              <div className="p-notification--negative">
+                <p className="p-notification__response">
+                  Hmm.. something has went wrong with this part of the application.
+                  If this problem persists, please <a
+                    className="p-link--external"
+                    href="https://github.com/CanonicalLtd/jujucharms.com">
+                    raise an issue</a> with the contents of the error message below.
+                  {error.message &&
```

Comment:
We should also have a default message as we're pointing them to an error message 'below'